app/components/ui/tale-tab-metadata/component.js

```diff
@@ -0,0 +1,68 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+import $ from 'jquery';
+
+import config from '../../../config/environment';
+
+const taleStatus = Object.create({
+  NONE: -1,
+  READ: 0,
+  WRITE: 1,
+  ADMIN: 2,
+  SITE_ADMIN: 100
+});
+
+
+export default Component.extend({
+  apiHost: config.apiHost,
+  environments: [],
+  model: null,
+  init() {
+    this._super(...arguments);
+    
+    // Fetch images for user to select an environment
+    let component = this;
+    $.getJSON(this.get('apiHost') + '/api/v1/image/').then(function(images) {
```

Comment:
With services: I can perhaps create a service for this single GET request, but it seems like overkill... Doesn't Ember Data provide a way for me to call query a RESTFUL resource? In this case, we aren't updating or deleting, simply listing all entries. 

With route models: Due to how the application is currently architected, we don't necessarily have access to the full `model` provided by `RSVP.hash` in the routes...

For example, assuming that (`app/templates/run/view.hbs`)[https://github.com/whole-tale/dashboard/blob/master/app/templates/run/view.hbs] is still in use, the `left-panel` child's `model` seem to be restricted to the parent's `model.selected`, while the `right-panel` child's `model` is restricted to the parent's `model.instances`. Adding a new value to the `RSVP.hash` in this case, does not allow the `left-panel` to access it. I can change it so that the `left-panel` has access to the full model instead, but that seems like a change we should discuss in case I'm unaware of the repercussions thereof.

Reply:
Pushed an example diff of changing the `$.getJSON` to instead use the Route model here: https://github.com/whole-tale/dashboard/compare/whole-tale:refactor-view-edit-tale-metadata...refactor-tale-metadata-with-route-model

Let me know if we think this might be a workable solution, I can merge those additional changes into this PR. If this change is too big or changes too many of our assumptions, then we can discuss further before merging it.